__tests__/integration/utils/wallet-helper.js

```diff
@@ -39,30 +53,87 @@ export class WalletHelper {
     return this.#addresses;
   }
 
+  get started() {
+    return this.#started;
+  }
+
   /**
-   * Creates a wallet object but does not start it on server
-   * @param {string} walletId
-   * @param {string} [words] 24 words
+   * Starts all the wallets needed for the test suite.
+   * <b>This is the preferred way of starting wallets</b> on the Integration Tests,
+   * performance-wise.
+   * @param {WalletHelper[]} walletsArr Array of WalletHelpers
+   * @param [options]
+   * @param {boolean} [options.skipAddresses] Skips the getSomeAddresses command
+   * @param {number} [options.amountOfAddresses=10] How many addresses should be cached per wallet
+   * @returns {Promise<void>}
    */
-  constructor(walletId, words) {
-    if (!walletId) throw new Error(`Wallet must have a walletId`);
-    this.#walletId = walletId;
+  static async startMultipleWalletsForTest(walletsArr, options) {
+    /**
+     * A map of `WalletHelper`s indexed by their `walletId`s
+     * @type {Record<string,WalletHelper>}
+     */
+    const walletsPendingReady = {};
 
-    this.#words = words || TestUtils.generateWords();
+    // If the genesis wallet is not instantiated, start it. It should be always available
+    const { genesis } = WALLET_CONSTANTS;
+    const isGenesisStarted = await TestUtils.isWalletReady(genesis.walletId);
+    if (!isGenesisStarted) {
+      walletsArr.unshift(new WalletHelper(genesis.walletId, genesis.words));
+    }
+
+    // Requests the start of all the wallets in quick succession
+    for (const wallet of walletsArr) {
```

Comment:
What happens if we do this in parallel?

Reply:
That is a good refactoring, but had no noticeable result on the test times.

Reply:
The `/start` API does almost nothing, I guess that's the reason it didn't increase performance. But I agree it's a good refactor